test/runtests.jl

```diff
@@ -1,5 +1,25 @@
 using Test, MixedLayerThermoclineDynamics
 
-@test hello("Julia") == "Hello, Julia"
-@test domath(2.0) ≈ 7.0
-@test MixedLayerThermoclineDynamics.compute_square(2.0) ≈ 4.0
+@time @testset "Grid tests" begin
+    include("test_grids.jl")
+
+    nx, ny = 10, 12
+    Lx, Ly = 2.0, 2.4
+    
+    grid1D = Grid1D(nx, 0, Lx)
+
+    @test test_dx(grid1D, Lx/nx)
+    @test test_xF(grid1D, range(0, stop = Lx - grid1D.dx, length = nx))
```

Comment:
Just a remark, here you are using `grid1D.dx` which assumes that indeed the grid was constructed with the correct `dx`. This is OK since in the previous line you tested that. Just wanted to point it out. It would be better to write a test that does not rely on other tests having tested things... E.g., a better way would be

```Julia
dx = Lx/nx

@test test_dx(grid1D, dx)
@test test_xF(grid1D, range(0, stop = Lx - dx, length = nx))`
...
```